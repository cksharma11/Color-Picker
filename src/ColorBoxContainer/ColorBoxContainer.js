import React from "react";
import ColorBox from "../components/ColorBox/ColorBox";

const ColorBoxContainer = ({ colors }) => {
  return (
    <div className="div-color-container">
      {colors.map(color => {
        return <ColorBox haxCode={color} />;
      })}
    </div>
  );
};

export default ColorBoxContainer;
