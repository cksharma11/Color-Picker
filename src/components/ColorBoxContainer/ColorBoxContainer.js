import React from "react";
import ColorBox from "../ColorBox/ColorBox";

const ColorBoxContainer = ({ colors }) => {
  return (
    <div className="div-color-container">
      {colors.map(color => {
        return <ColorBox haxCode={color} key={color} />;
      })}
    </div>
  );
};

export default ColorBoxContainer;
