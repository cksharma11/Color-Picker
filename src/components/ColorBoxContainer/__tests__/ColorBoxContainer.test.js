import React from "react";
import Enzyme, { shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import ColorBoxContainer from "../ColorBoxContainer";

Enzyme.configure({ adapter: new Adapter() });

describe("ColorBoxContainer", () => {
  it("should render properly", () => {
    const wrapper = shallow(<ColorBoxContainer colors={['red', 'green']} />);
    expect(wrapper.debug()).toMatchSnapshot();
  });
});
