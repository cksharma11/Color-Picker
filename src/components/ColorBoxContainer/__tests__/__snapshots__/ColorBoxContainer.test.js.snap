// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ColorBoxContainer should render properly 1`] = `
"<div className=\\"div-color-container\\">
  <ColorBox haxCode=\\"red\\" />
  <ColorBox haxCode=\\"green\\" />
</div>"
`;
